addons/im_livechat/static/src/embed/common/close_confirmation.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="im_livechat.closeConfirmation">
+        <div
+            class="o-livechat-closeConfirmation-overlay z-1 bg-black-50 position-absolute w-100 h-100 d-flex justify-content-center align-items-center"
+            t-on-keydown.capture.stop="onKeydown"
+            t-on-click.stop="onClick"
+            t-ref="closeConfirmation"
+            tabindex="1"
+        >
+            <div class="o-livechat-closeConfirmation-dialog rounded bg-white bg-opacity-100 p-3 m-3 d-flex flex-column">
+                <div class="d-flex justify-content-end">
+                    <button class="o-livechat-closeConfirmation-close btn-close" />
+                </div>
+                <span class="pt-2 pb-2">Are you sure you want to leave the current conversation?</span>
+                <button class="o-livechat-closeConfirmation-leave btn btn-primary p-2" title="Leave conversation">Leave conversation</button>
+            </div>
```

Comment:
I don't really like the wording of such confirm text.

`Are you sure you want to leave the current conversation?` is verbose and gives a vibe of danger.
I would prefer `You're about to leave the conversation, proceed?`, more to the point and less intimidating.

The `Leave conversation` label of button doesn't look like a confirmation at a quick glance. I would prefer `Yes, leave conversation`. This looks verbose but actually the "Yes, " helps a lot in quick and confident decision to confirm the closing or livechat.